packages/driver/types/cypress/log.d.ts

```diff
@@ -11,14 +11,14 @@ declare namespace Cypress {
   }
 
   type ReferenceAlias = {
```

Comment:
Not sure why these changed - all I did was run `yarn`. The changes are more in line with the style of the rest of the file, though, so I think it's okay.